### ISSUE TITLE
How-to connect to multiple SharePoint site collections in node using pnpjs?

### ISSUE THREAD

**Issue by Yvand** [pnp/pnpjs#1838]
### Category
- [ ] Enhancement
- [ ] Bug
- [x] Question
- [ ] Documentation gap/issue

### Version

Please specify what version of the library you are using:
@pnp/nodejs-commonjs: 2.7.0
@pnp/sp-commonjs: 2.7.0

Please specify what version(s) of SharePoint you are targeting: [   SharePoint Online      ]

### Expected / Desired Behavior / Question
I'm working on a NodeJS project that needs to connect to 2 SharePoint Online SPSites (same tenant, same client Id/secret).
Basically, it reads some data in the 1st SPSite and uses it to set data on the 2nd SPSite
What would be the recommended approach with Node to create 2 different connections?

I tried to create 2 SPRest objects but I don't know how to pass a unique "sp" object in setup() method.

Thank you for your help!

**Comment by juliemturner** [comment]
Please see the documentation on Isolated Runtimes: https://pnp.github.io/pnpjs/concepts/configuration/#isolated-runtimes

**Comment by Yvand** [comment]
It's exactly what I was looking for, it's too bad I was not able to find it by myself despite my research.
Thank you very much for your very quick help.
